docs/fate-points.md

```diff
@@ -0,0 +1,26 @@
+# The Fate Point Economy
+
+## Spending
+
+The primary usage of Fate Points is to [Invoke and Compel Aspects](aspects), but they have other uses.
+
+A Fate Point may be paid to Declare A Story Detail, such as establishing a relationship between you and a key NPC. Ideally, this should be rooted in one of your Aspects, but it is not an invocation since it cannot be done with a Free Invoke.
+
+Fate Points can also be the cost required to activate a Stunt. However, it is strongly recommended to avoid designing a Stunt with this cost; Fate Point refresh can be exchanged for Stunts, so the creation of a Stunt is like a permanently spent Fate Point. Requiring a Fate Point cost for a Stunt is thus equivalent to losing 2FP, so it better feel like it's worth spending those 2FP.
+
+Unlike vanilla Fate Core, Fate Points do not need to be spend to refuse Compels.
+
+## Replenishing
+
+Fate Points replenish automatically between sessions, *except* when continuing combat through multiple sessions.
+
+Otherwise, Fate Points can be earned in three ways:
+1. Accepting a Compel
+2. Have your Character Aspects and Consequences invoked against you with Fate Points
+3. Concede from a conflict
+
+You *can* earn more Fate Points than your refresh number! 
+
+## GM Fate Points
+
+Unlike players, who have a number of Fate Points for their character, a GM's Fate Points are shared between all NPCs.
```

Comment:
GMs also get fate points per scene, rather than per session, and gets fate points per PC in the scene.